package.json

```diff
@@ -39,27 +42,24 @@
   "files": [
     "dist"
   ],
-  "dependencies": {},
   "devDependencies": {
     "@event-driven-io/emmett": "0.1.6",
     "@faker-js/faker": "8.4.1",
     "@types/express": "4.17.21",
-    "@types/jest": "29.5.0",
     "@types/node": "20.11.17",
     "@types/uuid": "9.0.8",
     "@typescript-eslint/eslint-plugin": "6.21.0",
     "@typescript-eslint/parser": "6.21.0",
     "eslint": "8.56.0",
     "eslint-config-prettier": "9.1.0",
     "eslint-plugin-prettier": "5.1.3",
-    "jest": "29.7.0",
-    "npm-run-all": "4.1.5",
+    "npm-run-all2": "6.1.2",
```

Comment:
I decided to remove the [npm-run-all](https://www.npmjs.com/package/npm-run-all) library since it's not being maintained for 5 years now - the `npm-run-all2` one is a fork with dependabots updates (and some QOF enhancements as well)

Reply:
Fine by me 👌